# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../foo2zjs/no-lp-user-group.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- foo2zjs/Makefile	2005-03-17 15:05:30.000000000 +0100
+++ foo2zjs-fixed/Makefile	2005-06-03 08:57:54.000000000 +0200
@@ -20,8 +20,8 @@
 FOODB=/usr/share/foomatic/db/source
 
 # User ID's
-LPuid=-olp
-LPgid=-glp
+LPuid=-oroot
+LPgid=-groot
 ifeq ($(UNAME),Darwin)
     LPuid=-oroot
     LPgid=-gwheel
